### ISSUE TITLE
CLI --help and --version exit !=0 in 5.0.0

### ISSUE THREAD

**Issue by bollwyvl** [lovesegfault/beautysh#51]
Over on [conda-forge](https://github.com/conda-forge/beautysh-feedstock/pull/5) we had to temporarily break our "sanity check" testing for 5.0.0, as the behavior of `--help` (and `--version`) has changed:

```bash
jovyan@jupyter-jupyterlab-2djupyterlab-2ddemo-2dvsakdy2y:~$ beautysh --help
usage: beautysh [--indent-size INDENT_SIZE] [--backup] [--check] [--tab] [--force-function-style FORCE_FUNCTION_STYLE] [--version] [--help] FILE [FILE ...]
beautysh: error: the following arguments are required: FILE
```
```bash
jovyan@jupyter-jupyterlab-2djupyterlab-2ddemo-2dvsakdy2y:~$ beautysh --version
usage: beautysh [--indent-size INDENT_SIZE] [--backup] [--check] [--tab] [--force-function-style FORCE_FUNCTION_STYLE] [--version] [--help] FILE [FILE ...]
beautysh: error: the following arguments are required: FILE
```

I haven't dug into the source _too_ much, but it seems like this is probably a regression... `nargs="*"`, or special-casing `--help` and `--version` would be great.

**Comment by lovesegfault** [comment]
D'oh, I goofed when removing the `-f` flag. On it.

**Comment by lovesegfault** [comment]
@bollwyvl New release enroute

**Comment by lovesegfault** [comment]
@bollwyvl Released 5.0.1 with the fix!

**Comment by bollwyvl** [comment]
Huzzah, thanks for the insanely-quick turnaround!
